lib/pf/Switch/Template.pm

```diff
@@ -474,17 +474,18 @@ sub bouncePort {
         return $TRUE;
     }
 
-    return $self->_bouncePortCoa($ifindex);
+    my $radiusBounce = $self->{_template}{bounce};
+    if (!defined $radiusBounce) {
```

Comment:
Add `!isenabled($self->{_template}->{snmpDisconnect}` in the if

Reply:
If I add that in the if statement it would be a logical error.
Since I am expecting radiusBounce to be defined here https://github.com/inverse-inc/packetfence/pull/5735/files#diff-e7181847796bf1ef221178bfd4822927R483

Reply:
Yes, but if you enable snmpDisconnect and add bounce in the template configuration, the disconnection method will call handleReAssignVlanTrapForWiredMacAuth which uses bouncePort and then it will preform COA not SNMP.

Reply:
Thank you for the better explanation of the issue.

Reply:
To avoid the least surprise.
If you define bounce attributes we will use that when performing a bounce.
If you enable snmpDisconnect then we will use SNMP to disconnect.
It is the responsibility of the person configuring the Template Switch to choose the behavior they want.

I pushed the code to do this.